### PR TITLE
docs: fix duplicated desc

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |  -         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/icon/index.zh-CN.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 category: Components
 subtitle: 图标
 description: 语义化的矢量图形。
+customDesc: true
```
Comment (afc163) [inline components/icon/index.zh-CN.md:5]: 这个好难受。。。

File: components/icon/index.zh-CN.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 category: Components
 subtitle: 图标
 description: 语义化的矢量图形。
```
Comment (kiner-tang) [inline components/icon/index.zh-CN.md:4]: 没办法根据 markdown 当中有自定义的 description 自动识别吗？

File: components/icon/index.zh-CN.md (hunk 1 above)
Comment (MadCcc) [reply components/icon/index.zh-CN.md:5]: 其实可以 dumi 做，但自己也能加就是了。。这样是影响最小的做法

File: components/icon/index.zh-CN.md (hunk 2 above)
Comment (MadCcc) [reply components/icon/index.zh-CN.md:4]: dumi 自动取 markdown 的第一段话当 description，没有其他标记

File: components/icon/index.zh-CN.md (hunk 1 above)
Comment (afc163) [reply components/icon/index.zh-CN.md:5]: 让 dumi 解吧

**Review by li-jia-nan — APPROVED** [review]

File: .dumi/remarkAntd.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import yaml from 'js-yaml';
+
+let visit: typeof import('unist-util-visit').visit;
+
+// workaround to import pure esm module
+(async () => {
+  ({ visit } = await import('unist-util-visit'));
+})();
```
Comment (Wxh16144) [inline .dumi/remarkAntd.ts:8]: 这样使用算不算幽灵依赖了， 感觉可以直接从 dumi 导入 visit，和 `.dumi/rehypeAntd.ts` 类似

Comment (MadCcc) [reply .dumi/remarkAntd.ts:8]: 可以加一下

File: .dumi/remarkAntd.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+export default function remarkMeta() {
+  return (tree, vFile) => {
+    // read frontmatter
+    visit(tree, 'yaml', (node) => {
+      try {
+        const frontmatter = yaml.load(node.value) as any;
+        if (!('description' in frontmatter)) {
+          frontmatter.__autoDescription = true;
+        }
+        Object.assign(vFile.data.frontmatter!, frontmatter);
```
Comment (PeachScript) [inline .dumi/remarkAntd.ts:19]: 直接赋值 `vFile.data.frontmatter.__autoDescription` 即可？其他值 dumi 已经读到了

File: .dumi/remarkAntd.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+(async () => {
+  ({ visit } = await import('unist-util-visit'));
+})();
+
+export default function remarkMeta() {
+  return (tree, vFile) => {
+    // read frontmatter
+    visit(tree, 'yaml', (node) => {
+      try {
+        const frontmatter = yaml.load(node.value) as any;
```
[the remark this answers is not in the document]
Comment (MadCcc) [reply .dumi/remarkAntd.ts:9]: 这个可能会读到子节点？

[the remark this answers is not in the document]
Comment (PeachScript) [reply .dumi/remarkAntd.ts:9]: 有 `(^|[\r\n])` 就肯定是顶层节点了

File: .dumi/remarkAntd.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import { unistUtilVisit } from 'dumi';
+
+export default function remarkMeta() {
+  return (tree, vFile) => {
+    // read frontmatter
+    unistUtilVisit.visit(tree, 'yaml', (node) => {
+      try {
```
Comment (PeachScript) [inline .dumi/remarkAntd.ts:7]: try 也不需要了，正则匹配不会报错

**Review by PeachScript — APPROVED** [review]
+1

**Pull request merged**